### PR TITLE
wifi-menu: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/wifi-menu.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# wifi-menu
+
+> An easy to use interactive menu to setup network.
+> More information: <https://manned.org/wifi-menu>.
+
+- Interactively setup a connection to a network:
```
Comment (navarroaxel) [inline pages/linux/wifi-menu.md:6]: Perhaps?
```suggestion
- Setup a wireless connection interactively:
```

File: pages/linux/wifi-menu.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# wifi-menu
+
+> An easy to use interactive menu to setup network.
```
Comment (navarroaxel) [inline pages/linux/wifi-menu.md:3]: I think this is more descriptive:
```suggestion
> Interactively connect to a wireless network.
```

File: pages/linux/wifi-menu.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+> Interactively connect to a wireless network.
+> More information: <https://manned.org/wifi-menu>.
+
+- Setup a wireless connection interactively:
+
+`wifi-menu`
+
+- Interactively setup a connect to a network and obscure the password:
+
+`wifi-menu --obscure`
```
Comment (sbrl) [inline pages/linux/wifi-menu.md:12]: Wait why is this an option?! It should be enabled by default.

**Review by sbrl — APPROVED** [review]
Thanks, @CleanMachine1!

File: pages/linux/wifi-menu.md (hunk 3 above)
Comment (CleanMachine1) [reply pages/linux/wifi-menu.md:12]: Because who doesn't want someone peering over their shoulder at their network password

**Review by navarroaxel — APPROVED** [review]

**Review by SethFalco — APPROVED** [review]

File: pages/linux/wifi-menu.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# wifi-menu
+
+> Interactively connect to a wireless network.
+> More information: <https://manned.org/wifi-menu>.
+
+- Setup a wireless connection interactively:
```
Comment (marchersimon) [inline pages/linux/wifi-menu.md:6]:
```suggestion
- Set up a wireless connection interactively:
```

File: pages/linux/wifi-menu.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+`wifi-menu`
+
+- Interactively setup a connect to a network and obscure the password:
```
Comment (marchersimon) [inline pages/linux/wifi-menu.md:10]:
```suggestion
- Interactively set up a connection to a network and obscure the password:
```
```suggestion
- Interactively connect to a network and obscure the password:
```
Either are fine

**Review by marchersimon — CHANGES_REQUESTED** [review]

**Review by schneiderl — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
👍🏽

**Pull request merged**